src/test/java/org/reaktivity/specification/http/internal/FunctionsTest.java

```diff
@@ -56,11 +59,23 @@ public void headerTest()
         byte[] expected = new byte[] { (byte) 0x00, (byte) "name".length(), (byte) 'n', (byte) 'a', (byte) 'm', (byte) 'e',
                 (byte) "value".length(), (byte) 'v', (byte) 'a', (byte) 'l', (byte) 'u', (byte) 'e' };
```

Comment:
`byte[] expected` no longer needed, right?
Expectations are captured by asserting the flyweight fields match.